pyvelox/pyvelox.h

```diff
@@ -180,6 +222,37 @@ inline void appendVectors(VectorPtr& u, VectorPtr& v) {
   u->append(v.get());
 }
 
+inline VectorPtr evaluateExpression(
```

Comment:
This doesnt need to be inlined, also can we move this to the cpp file.

Reply:
Sure, why not move everything else to the cpp file too?

Reply:
Also I'm not really sure why everything was marked inline to begin with, it should all be `static`.

Reply:
Moved t hem